src/config/configuration.validator.ts

```diff
@@ -0,0 +1,23 @@
+import Ajv, { Schema } from 'ajv';
+
+const configurationSchema: Schema = {
+  type: 'object',
+  properties: {
+    AUTH_TOKEN: { type: 'string' },
+    EXCHANGE_API_KEY: { type: 'string' },
+  },
+  required: ['AUTH_TOKEN', 'EXCHANGE_API_KEY'],
+};
+
+export function validate(
+  configuration: Record<string, unknown>,
+): Record<string, unknown> {
+  if (process.env.NODE_ENV !== 'test') {
```

Comment:
This check prevents this environment variables validation to run while executing the tests. Having the full environment defined shouldn't be mandatory to run the application tests IMO.

Reply:
I would say that not having this check is also ok. When we start a test application it should be almost identical to the real one (minus some testing modules). But for now this is ok!